server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/OscService.java

```diff
@@ -102,6 +115,17 @@ public OscSwapTableVO swapTable(@PathVariable Long scheduleTaskId) {
         OnlineSchemaChangeParameters oscParameters = JsonUtils.fromJson(scheduleEntity.get().getJobParametersJson(),
                 OnlineSchemaChangeParameters.class);
 
+        Optional<FlowInstance> optional = flowFactory.getFlowInstance(oscParameters.getFlowInstanceId());
```

Comment:
we should dealloc flow instance when you finish using it

Reply:
okey, done